integration-tests/gradle/src/test/java/io/quarkus/gradle/IntegrationTestBuildTest.java

```diff
@@ -0,0 +1,20 @@
+package io.quarkus.gradle;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import java.io.File;
+
+import org.junit.jupiter.api.Test;
+
+public class IntegrationTestBuildTest extends QuarkusGradleWrapperTestBase {
+
+    @Test
+    public void shouldRunIntegrationTestAsPartOfBuild() throws Exception {
+        File projectDir = getProjectDir("it-test-basic-project");
+
+        BuildResult buildResult = runGradleWrapper(projectDir, "clean", "quarkusIntTest");
```

Comment:
See comment above re: should the `intTest` task trigger the `check` (or at a minimum `test`) task

Reply:
Right, I will add a dependency the `check` task.